stdlib/turtle.pyi

```diff
@@ -208,7 +208,7 @@ class TPen(object):
 _T = TypeVar("_T")
 
 class RawTurtle(TPen, TNavigator):
-    screens: list[ScrolledCanvas | Canvas | TurtleScreen]
+    screens: ClassVar[list[ScrolledCanvas | Canvas | TurtleScreen]]
```

Comment:
It always seems to be an instance of `TurtleScreen` (sometimes its subclass `_Screen`). If `ScrolledCanvas` or `Canvas` is passed to `RawTurtle.__init__`, it does `self.screen = TurtleScreen(canvas)` and then appends that to the `screens` list.